docker: invalid reference format: repository name must be lowercase.

For the past few days we have been experiencing the error `docker: invalid reference format: repository name must be lowercase.` when using php-actions/composer@master. Changing to @v6 resolves the issue.

I tracked the issue out of the action itself and off to https://raw.githubusercontent.com/php-actions/php-build/v2.2.0/php-build.bash and then got a headache looking through the shell script.

**My step:**
```
- uses: php-actions/composer@master
  with:
    php_version: 8.2
    php_extensions: sockets
    dev: yes
```

**Action output:**
```
Run php-actions/composer@master
  with:
    php_version: 8.2
    php_extensions: sockets
    dev: yes
    version: latest
    command: install
    interaction: no
    progress: no
    quiet: no
  env:
    COMPOSER_PROCESS_TIMEOUT: 0
    COMPOSER_NO_INTERACTION: 1
    COMPOSER_NO_AUDIT: 1
Run set -e
  set -e
  bash <(curl -s https://raw.githubusercontent.com/php-actions/php-build/v2.2.0/php-build.bash) composer
  /home/runner/work/_actions/php-actions/composer/master/composer-action.bash
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    COMPOSER_PROCESS_TIMEOUT: 0
    COMPOSER_NO_INTERACTION: 1
    COMPOSER_NO_AUDIT: 1
    ACTION_VERSION: latest
    ACTION_PHP_VERSION: 8.2
    ACTION_PHP_EXTENSIONS: sockets
    ACTION_TOKEN: ***
    ACTION_COMMAND: install
    ACTION_ONLY_ARGS: 
    ACTION_INTERACTION: no
    ACTION_DEV: yes
    ACTION_PROGRESS: no
    ACTION_QUIET: no
    ACTION_ARGS: 
    ACTION_SSH_KEY: 
    ACTION_SSH_KEY_PUB: 
    ACTION_SSH_DOMAIN: 
    ACTION_SSH_PORT: 
    ACTION_WORKING_DIR: 
    ACTION_MEMORY_LIMIT: 
    ACTION_CONTAINER_WORKDIR: 
docker: invalid reference format: repository name must be lowercase.
See 'docker run --help'.
Error: Process completed with exit code 125.
```

--- a/composer-action.bash
+++ b/composer-action.bash
@@ -196,7 +196,7 @@ do
 	fi
 done <<<$(env)
 
-if [ -z "$ACTION_CONTAINER_WORKDIR" ]; then
+if [ -n "$ACTION_CONTAINER_WORKDIR" ]; then
 	container_workdir="${ACTION_CONTAINER_WORKDIR}"
 fi
 

